README.md

```diff
@@ -1,12 +1,16 @@
-<p align="center"><img src="https://flarum.org/img/logo.png"></p>
+<p align="center"><img src="https://discuss.flarum.org/assets/logo-2q2rcevl.png"></p>
```

Comment:
Let's use https://flarum.org/assets/img/logo.png instead of this fingerprinted one, please.